croud/organizations/users/commands.py

```diff
@@ -39,6 +39,20 @@ def org_users_add(args: Namespace):
     client.print(keys=["user_id", "role_fqn", "organization_id"])
 
 
+@org_id_config_fallback
+def org_users_list(args: Namespace) -> None:
+    """
+    Lists organization users
+    """
+
+    client = Client.from_args(args)
+    client.send(RequestMethod.GET, f"/api/v2/organizations/{args.org_id}/users/")
+    if client._data and client._output_fmt in {"table"}:
+        for user in client._data:
+            user["role_fqn"] = user["organization_roles"][0]["role_fqn"]
+    client.print(keys=["uid", "email", "username", "role_fqn"])
```

Comment:
What do you think about changing the signature of `client.print()` to also allow a list of "transformer" functions for the specified keys:

```suggestion
    client.print(
        keys=["uid", "email", "username", "role_fqn"],
        transform=[identity, identity, identity, transform_org_role],
    )
    
def identity(key, item):
    return item[key]
    
def transform_org_role(key, item):
    return item["organization_roles"][0][key]
```

Reply:
Same applies for the other transformations.

Reply:
I like the idea.